src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs

```diff
@@ -2569,6 +2594,25 @@ public sealed partial class X509AuthorityInformationAccessExtension : System.Sec
         public System.Collections.Generic.IEnumerable<string> EnumerateUris(System.Security.Cryptography.Oid accessMethodOid) { throw null; }
         public System.Collections.Generic.IEnumerable<string> EnumerateUris(string accessMethodOid) { throw null; }
     }
+    public sealed partial class X509AuthorityKeyIdentifierExtension : System.Security.Cryptography.X509Certificates.X509Extension
+    {
+        public X509AuthorityKeyIdentifierExtension() { }
+        public X509AuthorityKeyIdentifierExtension(byte[] rawData, bool critical = false) { }
+        public X509AuthorityKeyIdentifierExtension(System.ReadOnlySpan<byte> rawData, bool critical = false) { }
+        public System.ReadOnlyMemory<byte>? KeyIdentifier { get { throw null; } }
```

Comment:
Is a "null" memory significantly different than an "Empty" one?

Reply:
`null` would mean it wasn't present in the extension (which is allowed).  Empty would mean it was present, and was a zero-length identifier (which isn't, really).

But making "not present" be "empty" is technically syntactically ambiguous and anyone using it in that state it's really "it wasn't here... what are you hoping to match against?"